spec/models/custom_attribute_spec.rb

```diff
@@ -10,8 +10,4 @@
   it "returns the value type of DateTime custom attributes" do
     expect(time_custom_attribute.value_type).to eq(:datetime)
   end
-
-  it "returns the value type of Fixnum custom attributes" do
-    expect(int_custom_attribute.value_type).to eq(:fixnum)
-  end
```

Comment:
See the commit regarding custom attributes, we should really find out if we're actually using `value_type` because it's really brittle to use class names instead of using interfaces such as expecting fixnum/bignum/integer types to respond to comparison operators.  CC @alongoldboim (see the commit for more details)